packages/features/tasker/tasks/crm/createCRMEvent.ts

```diff
@@ -120,13 +134,23 @@ export async function createCRMEvent(payload: string): Promise<void> {
           throw new Error(`Credential not found for credentialId: ${app.credentialId}`);
         }
 
+        const existingBookingReferenceForTheCredential = existingBookingReferences.find(
+          (reference) => reference.credentialId === crmCredential.id
+        );
+
+        if (existingBookingReferenceForTheCredential) {
```

Comment:
This is needed because  it is possible that only one of the CRM apps connected to the event failed and when we retry we don't know which one to retry again. So, we could just check if bookingReference exists already or not, if it exists we could just ignore that.